blocks/footer/footer.css

```
@@ -16,5 +17,134 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer a {
+  color: #fff;
+}
+
+footer li a:any-link {
+  color: white;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-container .right-side .icon{
+  fill: #949494;
+  transition: all 0.3s ease;
+  line-height: 1.6;
+  width: 32px;
+  height: 32px;
+  margin-bottom: 19px;
+}
+
+.footer-container [class^="footer-menu-list-"]{
+    padding-left:0;
+}
```

let's avoid `[class]`. We can use https://developer.mozilla.org/en-US/docs/Web/CSS/:is
```suggestion
.footer-container :is(.footer-menu-list-1, .footer-menu-list-2 ) {
  padding-left: 0;
  letter-spacing: 1px;
}

```